docs/content/latest/quick-start/build-apps/c/ysql.md

```diff
@@ -158,5 +164,154 @@ You should see the following output:
 ```output
 Created table employee
 Inserted data (1, 'John', 35, 'C')
-Query returned: John 35 C 
+Query returned: John 35 C
+```
+
+## Use C with SSL
+
+The client driver supports several SSL modes, as follows:
+
+| SSL mode | Client driver behavior |
+| :------- | :--------------------- |
+| disable | Supported |
+| allow | Supported |
+| prefer | Supported (default) |
+| require | Supported |
+| verify-ca | Supported |
+| verify-full | Supported |
+
+{{< note title="SSL mode support" >}}
+
+The default SSL mode for libpq is `prefer`. In the `require` SSL mode, the root CA certificate is not required to be configured.
+
+To enable `verify-ca` and `verify-full`, you configure the path to the CA in the connection string using the `sslrootcert` parameter. The default path is `~/.postgresql/root.crt`. If the root certificate is in a different file, specify it in the sslrootcert parameter. For example:
```

Comment:
I got error "Connection to database failed: root certificate file "/Users/myuser/.yugabytedb/root.crt" does not exist" which implies .postgresql is not the default directory